lib/MusicBrainz/Server/WebService/JSONSerializer.pm

```diff
@@ -54,6 +54,8 @@ sub work_list          { shift->entity_list(@_, "work", "works") };
 sub area_list          { shift->entity_list(@_, "area", "areas") };
 sub place_list         { shift->entity_list(@_, "place", "places") };
 sub event_list         { shift->entity_list(@_, "event", "events") };
+sub instrument_list    { shift->entity_list(@_, "instrument", "instruments") };
+sub series_list        { shift->entity_list(@_, "series", "series") };
```

Comment:
Ideally, this should be generated programmatically. If you don’t want to do that now, please leave a comment like `# FIXME: use entities_with` so that it can be found later.